### ISSUE TITLE
spaces around operators

### ISSUE THREAD

**Issue by steveklabnik** [stevedonovan/gentle-intro#9]
In section 1 at least

     x*x

most people write

    x * x

a minor thing but I figured I'd mention it!

**Comment by stevedonovan** [comment]
Yes, I generally prefer letting operators breathe as well

**Comment by eddyb** [comment]
Lack of space after some commas probably applies as well.

**Comment by stevedonovan** [comment]
Yes, I've also put space after the comma after the format string in `println`.  I'll keep the issue open since breathing is important ;)
